distributed/scheduler.py

```diff
@@ -4135,12 +4138,14 @@ def reevaluate_occupancy(self, worker_index=0):
             if self.status == 'closed':
                 return
 
-            import psutil
-            proc = psutil.Process()
             last = time()
             next_time = timedelta(seconds=DELAY)
 
-            if proc.cpu_percent() < 50:
+            # scheduler was somehow moved to another PID
+            if self.proc.pid != os.getpid():
+                self.proc = psutil.Process()
```

Comment:
You observed this happening?  This seems surprising ...

Reply:
I have not. Just a precaution that is likely unnecessary. I was thinking of running the scheduler on an HPC where it can be interrupted and resumed. Looks like that might break other parts of the code though.